docs/.vitepress/theme/custom.css

```diff
@@ -25,6 +28,10 @@
   font-weight: 500;
 }
 
+.vp-doc p {
+  line-height: 1.5;
```

Comment:
This should probably be on .vp-doc rather than just p? Otherwise things like lists will have a different line-height which will look weird.

Reply:
For me this is correct.

The default line-height on the body is set in `node_modules/vitepress/dist/client/theme-default/styles/base.css` as `24px`, but it's overloaded in
`node_modules/vitepress/dist/client/theme-default/styles/components/vp-doc.css` with the selector `.vp-doc p` (also `.vp-doc h1`, etc.). Resetting this on `.vp-doc` would not have enough specificity, and `.vp-doc p` is the only one we want to override.

Reply:
That seems like a bizarre out-of-the-box design by VitePress, but indeed you are right, the line-height is 24px for ul elements and 28px for p elements. It seems that VitePress avoids this problem on their own site by using p elements _within_ the li elements, restoring the 28px line-height. So, perhaps the intent is that you only use the 24px line-height for lists when you know that the list items all fit on a single line. 🤔 Anyway it means this is “correct” according to the weirdness of the VitePress default theme I guess.

Reply:
Agree it's more "correct" than it is correct :-)